keyboards/ergodox/keymaps/333fred/Makefile

```diff
@@ -1,7 +1,7 @@
 SUBPROJECT_DEFAULT = infinity
 LCD_BACKLIGHT_ENABLE = yes
 LCD_ENABLE = yes
-LED_ENABLE = yes
+BACKLIGHT_ENABLE = yes
```

Comment:
I pinged @fredizzimo on the gitter, but haven't heard back. It doesn't appear to be working at the moment.